src/lumigo_tracer/wrappers/http/http_parser.py

```diff
@@ -234,6 +234,38 @@ def parse_response(self, url: str, status_code: int, headers, body: bytes) -> di
         )
 
 
+class EventBridgeParser(Parser):
+    def parse_request(self, parse_params: HttpRequest) -> dict:
+        try:
+            parsed_body = json.loads(parse_params.body)
+        except json.JSONDecodeError as e:
+            get_logger().debug("Error while trying to parse eventBridge request body", exc_info=e)
+            parsed_body = {}
+        resource_names = set()
+        if isinstance(parsed_body.get("Entries"), list):
+            resource_names = {
+                e["EventBusName"] for e in parsed_body["Entries"] if e.get("EventBusName")
+            }
+        return recursive_json_join(
+            {"info": {"resourceName": ",".join(resource_names) or None}},
```

Comment:
This is our way to pass lists to the backend? cant we use array?

Reply:
So that's actually a string - I don't want the BE to handle it as a list 🤔

Reply:
Its a list of all the event buses, you just make it a string by joining with `,`
Better option: pass another field `resourceNames` to the backend, the make this join over there when building the span.
The main issue here, its if we will want to show that in another way to our users (as a list for example), we will not able to do-it because this code in our tracer can't be really change (we will need all ways backward compatible)

Reply:
Awesome idea!
Will do!